projects/samples/devices/controllers/console_capabilities/ansi_codes.h

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 1996-2020 Cyberbotics Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef ANSI_CODES_H
+#define ANSI_CODES_H
+
+#include <stdio.h>
+
+#define ANSI_STYLE_RESET "\x1b[0m"
+
+#define ANSI_BOLD_STYLE "\x1b[1m"
+#define ANSI_UNDERLINE_STYLE "\x1b[4m"
+
+#define ANSI_COLOR_BG_BLACK "\x1b[40m"
```

Comment:
We try to avoid abbreviations:
```suggestion
#define ANSI_BLACK_BACKGROUND "\x1b[40m"
```
etc.